.github/workflows/generate-token-lists.yml

```diff
@@ -221,7 +221,9 @@ jobs:
             # Backup online list to {version}/{path} before deploying a new one
             lines=$(aws s3 ls s3://${{ secrets.AWS_BUCKET }}/$additionalPath$path | wc -l)
             if (( $lines > 0 )); then
-              aws s3 cp s3://${{ secrets.AWS_BUCKET }}/$additionalPath$path s3://${{ secrets.AWS_BUCKET }}/$additionalPath${{ steps.onlineVersion.outputs.onlineVersion }}/
+              backupCommand="aws s3 cp s3://${{ secrets.AWS_BUCKET }}/$additionalPath$path s3://${{ secrets.AWS_BUCKET }}/$additionalPath"
```

Comment:
https://github.com/OffchainLabs/arbitrum-token-lists/actions/runs/8052421755/job/21992515216?pr=131#step:7:34

Updated, it backup to `/TestFolder/ArbTokenLists/42170_arbed_gemini_token_list/2.0.0.json` now

Reply:
<img width="653" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-lists/assets/13184582/f2bd600c-554d-4f89-813e-2cbc3c3517ba">
that looks odd to me
why are there quotation marks?

Reply:
Github add quotation because the variable is coming from github environment, not the bash environment itself. But it does work:

> TestFolder/ArbTokenLists/42170_arbed_gemini_token_list/2.0.0.json